apps/system/utils/netcmd_netmon.c

```diff
@@ -176,10 +201,19 @@ int cmd_netmon(int argc, char **argv)
 			printf("Failed to fetch socket info.\n");
 		}
 	} else if (!(strncmp(argv[1], "wifi", strlen("wifi")))) {
```

Comment:
strlen("wifi") + 1 instead of strlen("wifi").

Reply:
Modification completed.